core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.ts

```diff
@@ -47,6 +48,15 @@ export class SideNavigationBarComponent {
 
    ngOnInit(): void {
      this.currentUrl = this.windowRef.nativeWindow.location.pathname;
+
+     // To stop catching of click event in bubbling phase,
+     // we use stopPropagation method.
+     this.listSubmenu = document.querySelector(
```

Comment:
Would this prevent propagation on clicking any other elements in the sidebar?

Reply:
No, currently we have only one expandable item in side bar, so it's applicable only there.

Explanation:-
just to pervent from closing of side-bar when we click on classroom, i attached `stopPropagation` so that click event unable to reach `togglesidebar()` which closes it every time.